scripts/w3.mjs

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
+
+// A tool acting like w3 CLI which enables executing a command with the given UCAN key and delegation proof.
+// To install the dependencies, execute the following command
+// npm i multiformats @ipld/car@5.1.0 @ucanto/principal@4.2.3 @web3-storage/w3up-client@4.1.0
+
+import fs from "fs";
+import { CID } from "multiformats";
+import { CarReader } from "@ipld/car";
+import { derive } from "@ucanto/principal/ed25519";
+import { importDAG } from "@ucanto/core/delegation";
+import { AgentData } from "@web3-storage/access";
+import { Client } from "@web3-storage/w3up-client";
```

Comment:
Seems like you need a bunch of external dependencies as well, so I'd say it's ideal to have its own `package.json` etc. You can move it to a sub-folder under this repo and have it setup under there for example.

Reply:
Good idea, moved.